Piwik.Tracker/PiwikTracker.cs

```diff
@@ -721,11 +742,11 @@ public TrackingResponse DoTrackEcommerceCartUpdate(double grandTotal)
         /// Response
         /// </returns>
         /// <exception cref="System.InvalidOperationException">Error: you must call the function DoTrackPageView or DoTrackGoal from this class, before calling this method DoBulkTrack()</exception>
-        public TrackingResponse DoBulkTrack()
+        public async Task<TrackingResponse> BulkTrackAsync()

```

Comment:
I am having difficulties understanding why the keywoard "async" is sometimes used and sometimes omitted in method declarations.

Could you help me understand maybe by providing a link?